### ISSUE TITLE
`print-version` missing in v8

### ISSUE THREAD

**Issue by shooit** [python-semantic-release/python-semantic-release#638]
## The problem

v8.x no longer supports the `print-version` command or another path to get the current version of the project

## Expected behavior

We have been using the `print-version` command to determine if a release was made in our CI processes. For example,

```yaml
jobs:
  release:
    runs-on: ubuntu-latest
    concurrency:
      group: release
      cancel-in-progress: false
    steps:
      - id: pre-release
        name: get pre-release version
        run: echo "version=$(poetry run semantic-release print-version --current)" >> $GITHUB_OUTPUT
      - name: release
        run: poetry run semantic-release publish
        env:
          GH_TOKEN: ${{ secrets.GH_TOKEN }}
      - id: post-release
        name: get post-release version
        run: echo "version=$(poetry run semantic-release print-version --current)" >> $GITHUB_OUTPUT
      - if: steps.pre-release.outputs.version != steps.post-release.outputs.version
        run: my-post-release-jobs
```

**Comment by bernardcooke53** [comment]
Hey @shooit - have you tried the following:
```bash
semantic-release --noop version --print
```

The [`--print`](https://python-semantic-release.readthedocs.io/en/latest/commands.html#print) option is new in v8.
It looks like you're using GitHub Actions and running `semantic-release` directly to set some outputs up for the workflow - just want to also mention that the GitHub Action [now has outputs](https://python-semantic-release.readthedocs.io/en/latest/migrating_from_v7.html#python-semantic-release-github-action):

* `released`, which looks a lot like your condition `steps.pre-release.outputs.version != steps.post-release.outputs.version`
* `version`, which is what you're setting as output when doing `poetry run semantic-release`

so you might get some mileage/simplification out of that too.

**Comment by bernardcooke53** [comment]
Ah, I see - there's no `--current` flag in the latest version. The main driving factor for that being that in v7, the version would be determined from source file and commit+tag history. We don't look at commits or source files to determine versions in v8. So the "current" version should just be whatever's stored in the locations you've configured - you can pull it out of the file directly.

For your use case there's a few useful one-liners with different tools:
* [toml-cli](https://crates.io/crates/toml-cli)
```bash
toml get pyproject.toml project.version
```
* pip + tomli
```bash
pip install tomli && python3 -c "import tomli; print(tomli.load(open('pyproject.toml'))['project']['version'])"
```
* bash
```bash
cat pyproject.toml | grep "version *=" | sed -r 's/^version *= *"(.*)"$/\1/'
```

Possibly a useful feature would be a new command, `list-versions`, so that you could do:

```bash
semantic-release list-versions -n 1
```

to get the latest version. What do you think?

**Comment by cfxegbert** [comment]
Printing the current version is addressed in pull request https://github.com/python-semantic-release/python-semantic-release/pull/852.  You can choose to print the last released version number or tag.

**Comment by codejedi365** [comment]
Released with v9.2.0
